src/store/project.ts

```diff
@@ -82,6 +82,16 @@ export class Project {
 		return this.id;
 	}
 
+	// TODO
+	public getLastChangedAuthor(): string {
```

Comment:
We don't have any user management at the moment and afaik there is no plan to implement one while we're still on alpha. Based on that I would remove this function as it's now only dead code.